tasks/bosh-create-env.sh

```diff
@@ -27,7 +27,7 @@ pushd state/environments/softlayer/director/$BOSH_LITE_NAME
     export DIRECTOR_URL="https://$(cat hosts | cut -f 2 -d ' '):25555/"
     export CA_CERT="$(bosh2 interpolate vars.yml --path /director_ssl/ca)"
     export CLIENT_SECRET="$(bosh2 interpolate vars.yml --path /admin_password)"
-    envsubst '${BOSH_LITE_NAME} ${DIRECTOR_URL} ${CA_CERT} ${CLIENT_SECRET}' < $envrc_template_filename > .envrc
+    envsubst '${BOSH_LITE_NAME} ${DIRECTOR_URL} ${CA_CERT} ${CLIENT_SECRET} ${DOMAIN_NAME}' < $envrc_template_filename > .envrc
```

Comment:
It seems `DOMAIN_NAME` is not used in the envrc template. Am I missing something?

Reply:
Fixed with the latest commit.